Update audit-logs.md (#6931)
cloudflare/cloudflare-docs: content/cloudflare-one/analytics/logs/audit-logs.md

### Non-identity authentication

> Requests blocked by a identity policies appear in the Zero Trust dashboard. To retrieve logs for non-identity authentication events, use the [GraphQL Analytics API](/analytics/graphql-api/tutorials/querying-access-login-events/).

Reviewer: ```suggestion
To retrieve logs for non-identity authentication events, use the [GraphQL Analytics API](/analytics/graphql-api/tutorials/querying-access-login-events/). Non-identity logs cannot be viewed on the Zero Trust dashboard. 
```
